tailwind.config.ts

```diff
@@ -4,6 +4,7 @@ import type { Config } from "tailwindcss";
 const config: Config = {
   content: ["./app/**/*.{ts,tsx}", "./packages/**/*.{ts,tsx}"],
   plugins: [daisyuiPlugin],
+  darkMode: "class",
```

Comment:
This should not be part of the current PR, this is related to the #184

Reply:
change here [7864ad4](https://github.com/podcodar/webapp/pull/198/commits/7864ad4c2075b4cb5fd36107208cdef3773f639b)